packages/@lwc/integration-karma/scripts/karma-configs/hydration/base.js

```diff
@@ -98,7 +98,7 @@ module.exports = (config) => {
         config.plugins.push('karma-coverage');
 
         config.coverageReporter = {
-            dir: path.resolve(COVERAGE_DIR, 'hydration'),
+            dir: path.resolve(COVERAGE_DIR, 'hydration', COVERAGE_DIR_FOR_OPTIONS),
```

Comment:
Our coverage for hydration tests was not being calculated correctly beforehand. Now it is.